Docs updates for Bitbucket Server SCM Flow (#500)
algorithmiaio/dev-center: _pages/algorithmia-enterprise/scms.md

<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/admin_scms/browse_scms.png" alt="Browse SCMs" class="screenshot img-sm">

> ### Creating a SCM

Reviewer: ```suggestion
### Creating an SCM
```

These permissions are distinct from those of the creating user–this ensures that, if that individual should leave your organization, Algorithmia will be able to continue to build new versions of the algorithm they created.

#### Create an OAuth App (Github, Gitlab, Bitbucket Cloud)

> The first step in configuring an OAuth based SCM is creating an OAuth application. An OAuth application allows individual users to authorize Algorithmia to act on their behalf. We securely store any authorization obtained, and only use it when necessary (such as when creating a new Github repository for an algorithm).

Reviewer: ```suggestion
The first step in configuring an OAuth-based SCM is creating an OAuth application. An OAuth application allows individual users to authorize Algorithmia to act on their behalf. We securely store any authorization obtained, and only use it when necessary (such as when creating a new Github repository for an algorithm).
```

- OAuth Application’s Client Secret (Keep this confidential at all times)
- GitHub URL (https://github.com for public GitHub, or a custom URL for GitHub Enterprise)

#### Create Bitbucket Server Based Token

> The first step in configuring a Bitbucket Server based token is creating a Personal Access Token. A Personal Access Token is used to authorize Algorithmia to act on their behalf. We securely store any authorization obtained, and only use it when necessary (such as when creating a new Github repository for an algorithm).

Reviewer: ```suggestion
The first step in configuring a Bitbucket Server-based token is creating a personal access token. A personal access token is used to authorize Algorithmia to act on their behalf. We securely store any authorization obtained, and only use it when necessary (such as when creating a new Github repository for an algorithm).
```

Next obtain the Project Token for the project that will be used for the new Bitbucket Server SCM you are creating.

#### Create an SCM

> With your Github OAuth application or Bitbucket Server Tokens created, it’s time to return to your Algorithmia instance.

Reviewer: ```suggestion
With your Github OAuth application or Bitbucket Server tokens created, it’s time to return to your Algorithmia instance.
```

__SSH URL__: This is the URL by which we will attempt to pull source code from your GitHub Instance. For public GitHub, this value should be ssh://git@github.com. For GitHub Enterprise instances, replace the domain name with the domain for your instance. For example, if your GitHub Enterprise instance lives at https://example.com, you should enter ssh://git@example.com. 

__OAuth Client ID__: This is the value of the “Client ID” you received during the “Create an OAuth App” step. 

__OAuth Client Secret__: This is the value of the “Client Secret” you received during the “Create an OAuth App” step. 

> In order to create a Bitbucket Server SCM, you must create a Personal Access Token in your Bitbucket Server instance and provide the Project Key of the project where you'd like repositories created. These fields replace the OAuth Client ID and OAuth Client Secret fields in the creation form when Bitbucket Server SCM is chosen from the dropdown:

Reviewer: ```suggestion
In order to create a Bitbucket Server SCM, you must create a personal access token in your Bitbucket Server instance and provide the project key of the project where you'd like repositories created. These fields replace the OAuth Client ID and OAuth Client Secret fields in the creation form when Bitbucket Server SCM is chosen from the drop-down:
```

> __Personal Access Token__: The token used by Algorithmia to create a repository for algorithms create a webhook for receiving repository events.

Reviewer: ```suggestion
__Personal Access Token__: The token used by Algorithmia to create a repository for algorithms to create a webhook for receiving repository events.
```

Reviewer: was this the intended meaning?
